_includes/head.html

```diff
@@ -8,7 +8,7 @@
 
   {% assign custom_url = site.url | append: site.baseurl %}
   {% assign full_base_url = custom_url | default: site.github.url %}
-  <link rel="stylesheet" href="{{ "/css/main.css" | prepend: full_base_url }}">
+  <link rel="stylesheet" href="{{ "/assets/css/main.css" | prepend: full_base_url }}">
```

Comment:
If we require Jekyll `~> 3.3`, I think we can make this a non-breaking change. There are two possibilities:

1. User overrode `head.html` to hard code `/css/main.css` and has a `/css/main.scss` already in their site (from `jekyll new`)

2. User has default head, will have a `/css/main.scss` file but it won't do anything. New head will use `/assets/style.css` which we will provide.

Having `/assets/style.scss` write to `/css/main.css` feels funny, especially because this is the default/starter site, and I suspect most users wouldn't know to look in `/assets` to replace a file in outputted to `/css` which is already confusing enough.